<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>regit</class>
 <widget class="QDialog" name="regit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>379</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="id_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>아이디</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="idbar">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>id</string>
   </property>
  </widget>
  <widget class="QLabel" name="pw_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>67</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>비밀번호</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="pwbar1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>pw</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="pwbar2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>pw</string>
   </property>
  </widget>
  <widget class="QPushButton" name="regit_button">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>340</y>
     <width>101</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>회원가입</string>
   </property>
  </widget>
  <widget class="QLabel" name="pw_label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>비밀번호 확인</string>
   </property>
  </widget>
  <widget class="QLabel" name="regit_label">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>0</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>회원가입</string>
   </property>
  </widget>
  <widget class="QPushButton" name="id_chk">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>50</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>중복확인</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="namebar">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>name</string>
   </property>
  </widget>
  <widget class="QLabel" name="pw_label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>67</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>이름</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="student">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>310</y>
     <width>112</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>student</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="teacher">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>310</y>
     <width>112</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>teacher</string>
   </property>
  </widget>
  <widget class="QLabel" name="pw_label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>280</y>
     <width>67</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>구분</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
